delta/plugins/storage/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/storage/storages/model/StorageValue.scala

```diff
@@ -116,11 +115,11 @@ object StorageValue {
       name: Option[String],
       description: Option[String],
       default: Boolean,
-      algorithm: DigestAlgorithm,
       bucket: String,
       readPermission: Permission,
       writePermission: Permission,
-      maxFileSize: Long
+      maxFileSize: Long,
+      algorithm: DigestAlgorithm = DigestAlgorithm.default
```

Comment:
I would even put it as a constant like the storage type

Reply:
When I put it as a constant it didn't serialize correctly. I will look into it